.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     container:
-      image: registry.gitlab.com/finestructure/spi-base:0.6.2
+      image: registry.gitlab.com/finestructure/spi-base:22eec776e982b298334fd9ed809fa2c78d0125b1
```

Comment:
This is referencing a new base image with sqlite3 libs, required to build the package collection module of `SwiftPM`.